pages/linux/pvcreate.md

```diff
@@ -0,0 +1,11 @@
+# pvcreate
+
+> Initialize a disk or partition for use by LVM (Logical Volume Manager).
+
+- Initialize /dev/sda1 to use lvm:
```

Comment:
/dev/sda1 should be in backticks. Sorry missed out this one in the previous review.